platforms/python/prereqs/build.sh

```diff
@@ -59,6 +59,7 @@ if [ "$debianFlavor" == "stretch" ]; then
 
     PIP_VERSION="20.2.3"
 else
+    PIP_VERSION="21.2.4"
```

Comment:
we can remove the debian-flavor = stretch condition entirely

Reply:
Will take this one in next pr